### ISSUE TITLE
openBox takes ~30 seconds to complete

### ISSUE THREAD

**Issue by oznekenzo** [3box/3box-js#292]
This seems to be occuring on profiles with lots of activity (Kenzo & Michaels)

**Comment by oznekenzo** [comment]
@oed Yeah this is always the case for me.  Let me know how I can provide more data if needed

**Comment by oed** [comment]
Thanks @oznekenzo. 
@zachferland is this something you think you'll be able to test with you new testing environment?
